SqlScriptDom/Parser/TSql/CodeGenerationSupporter.cs

```diff
@@ -1263,7 +1264,7 @@ internal static class CodeGenerationSupporter
         internal const string Russian = "RUSSIAN";
         internal const string Romanian = "ROMANIAN";
         internal const string Brazilian = "BRAZILIAN";
-        internal const string NorwegianBokmal = "NORWEGIAN (BOKM�L)";
+        internal const string NorwegianBokmal = "NORWEGIAN (BOKM�L)";
```

Comment:
change expected?

Reply:
nope, investigating the git setting that caused this

Reply:
file encoding was utf8, which is sensitive to messing up the combined bytes of some characters. the most definitive fix was to convert the file to utf8 bom file encoding, which is used by a number of other c# files in the project already